content/installations/install-node/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Installing Node"
+date: 2021-10-01T09:28:27-05:00
+draft: false
+weight: 5
+---
+
```

Comment:
Suggestion - be very specific about which node version they should install - as upgrading to a new version may cause security issues.